infra/ansible/e2e_tests.Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG python_version=3.8
+ARG debian_version=buster
+
+FROM python:${python_version}-${debian_version}
+
+WORKDIR /ansible
+RUN pip install ansible
+COPY . /ansible
+
+ARG ansible_vars
+
+# Build and install PyTorch and PyTorch/XLA wheels.
+RUN ansible-playbook -vvv playbook.yaml -e "stage=build" -e "${ansible_vars}"
```

Comment:
It looks like the test image here includes all of the build artifacts, which blows the size up to `7.1 GB` compared to `1.4 GB` in the current test image. Is it possible to use the image build we use for nightly releases? The current `nightly_3.8_tpuvm` is only 3 GB (we can optimize the size further in a follow-up).

Large images increase the run time of the CI and contribute to flakiness as the disk on the test host machines fill up. Auto-repair is unfortunately not available for our GKE setup right now.

Reply:
I'll try to optimise it. Running a new build & test: https://pantheon.corp.google.com/cloud-build/builds;region=us-central1/133b06ba-ed73-48a7-9fdb-6393565aa99f?project=tpu-pytorch